python/cuvs_bench/cuvs_bench/tests/test_modularization.py

```
@@ -0,0 +1,1002 @@
+        yield
+
+    def test_elastic_plugin_loads(self):
+        """Elastic backend and config loader load when elasticsearch is installed."""
+        assert get_backend_class("elastic") is not None
+        assert get_config_loader("elastic") is not None
+
+    def test_elastic_config_loader_tune_mode_returns_single_config(self):
+        """Tune mode produces one BenchmarkConfig with Optuna-suggested params."""
+        loader_cls = get_config_loader("elastic")
+        loader = loader_cls()
+        dataset_config, benchmark_configs = loader.load(
```

Ruff (0.15.15)

[warning] Unpacked variable `dataset_config` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)